File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/tabuada.c
#include <stdio.h>

int main() {
    int N, produto;

    printf("Deseja a tabuada para qual valor?");
    scanf("%d", &N);

    for (int i = 1; i <= 10; i++) {
        produto = N * i;
        printf("%d * %d = %d\n", N, i, produto);
    }

    return 0;
}

File: dev-superior-algoritmo-e-logica-de-programacao/java/src/exercicios/Tabuada.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class Tabuada {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int N, produto;

		System.out.print("Deseja a Tabuada para qual valor? ");
		N = sc.nextInt();

		for (int i = 1; i <= 10; i++) {
			produto = N * i;
			System.out.printf("%d x %d = %d\n", N, i, produto);
	    }

		sc.close();
	}
}
